hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestDynamoDBMetadataStoreScale.java

```diff
@@ -172,6 +174,23 @@ public void setup() throws Exception {
 
   @Override
   public void teardown() throws Exception {
+    if (ddbms != null) {
+      S3GuardTableAccess tableAccess = new S3GuardTableAccess(ddbms);
+      ExpressionSpecBuilder builder = new ExpressionSpecBuilder();
+      builder.withKeyCondition(
```

Comment:
use withCondition here instead of key, so you don't have to filter for the VersionMarker